app/actors/curation_concerns/actors/add_to_work_actor.rb

```diff
@@ -22,13 +22,17 @@ def add_to_works(new_work_ids)
             work.save
           end
 
-          # add to new
+          # add to new so long as the depositor for the parent and child matches, otherwise inject an error
           (new_work_ids - curation_concern.in_works_ids).each do |work_id|
             work = ::ActiveFedora::Base.find(work_id)
-            work.ordered_members << curation_concern
-            work.save
+            if work.depositor != curation_concern.depositor
```

Comment:
This line may come back to bite us, especially for folks with use cases where all depositors (in effect) share ownership of all objects. Thinking here of how Princeton and IU use CC. Ping @tpendragon @escowles 

I wonder if we should do an ability check here (delegating this responsibility to our Ability models and underlying hydra-access-controls code that knows how to inspect `edit_users` and `edit_groups`) instead of comparing depositors.